### DISCUSSION TITLE
undefined method `vite_javascript_tag' on staging environment

### DISCUSSION THREAD

**Discussion by rubyandcoffee** [ElMassimo/vite_ruby#347] [Q&A] [answered]
**Environment:** I'm on Rails 6 using the `vite_rails` gem

**Issue:**
Locally, Vite works fine. 
But when I push it to my staging environment, I get this error: `undefined method 'vite_javascript_tag'`

Any ideas?

**Comment by ElMassimo** [comment, marked as answer]
Hi Alexandra!

The error suggests that the [`TagHelpers`](https://github.com/ElMassimo/vite_ruby/blob/main/vite_rails/lib/vite_rails/tag_helpers.rb) are not available in your application.

Usually, tag helpers are loaded automatically in the [railtie included in `vite_rails`](https://github.com/ElMassimo/vite_ruby/blob/main/vite_rails/lib/vite_rails/engine.rb#L12), which should only happen if `vite_rails` is not being required.

For example if `gem "vite_rails"` is inside `group :development`, it would be installed locally, but it wouldn't be installed nor required in the `staging` environment.

Feel free to share your `Gemfile`.
